relay-metrics/src/aggregation.rs

```diff
@@ -2007,6 +2045,14 @@ mod tests {
         fn bucket_count(&self) -> usize {
             self.data.read().unwrap().buckets.len()
         }
+
+        async fn bucket_count_async(&self) -> usize {
+            // We have to wait for a bit to let the test receiver to be called once to receive the
+            // message.
+            relay_test::block_fn(|| relay_test::delay(Duration::from_millis(10)).map_err(|_| ()))
```

Comment:
Do we still need this now that we have a utility that runs the actix system alongside tokio?